lib/include/jxl/decode.h

```diff
@@ -403,7 +405,7 @@ JXL_EXPORT JxlDecoderStatus JxlDecoderSubscribeEvents(JxlDecoder* dec,
  * @return JXL_DEC_SUCCESS if no error, JXL_DEC_ERROR otherwise.
  */
 JXL_EXPORT JxlDecoderStatus
-JxlDecoderSetKeepOrientation(JxlDecoder* dec, JXL_BOOL keep_orientation);
+JxlDecoderSetKeepOrientation(JxlDecoder* dec, JXL_BOOL keep_data_orientation);
```

Comment:
Also have to update the `@param` name in the documentation above, those names need to match.

Reply:
I'm not sure if `keep_data_orientation` is a clearer name than `keep_orientation`. The word "data" is quite generic.

Maybe `skip_auto_orientation` would be a better name here. If you go just by the param name (ignoring the rest of the documentation), it is somewhat ambiguous what "keeping" orientation means: does it mean keeping the image un-oriented, i.e. not applying the orientation (this is what it means), or does it mean 'keeping' the image orientation, i.e. applying orientation (this is what it doesn't mean, but it could be misinterpreted easily this way).

Reply:
Agree that that name is not great either. If going further astray than merely expanding the name is OK, I would propose "ignore_reorientation" - it's short, and captures what is happening here. Perhaps: ignore_reorientation_step.

What do you think?

Reply:
I like `skip` more than `ignore`, I have no preference on `orientation` vs `reorientation` vs `auto_orientation`.

Since the function name cannot be changed, I'm also fine with sticking with `keep_orientation` (to keep it the same as in the function name) and just hoping that people will read the documentation.

Reply:
The problem is: "keep which orientation" - the one that the raw image data uses or the one that the orientation setting specifies?

Assuming that changing the function name is not an option here: "keep_data_orientation" would at least provide some more context by saying "it's the orientation that the data comes in that we want to keep here" - as opposed to using the orientation setting provided separately. And in that light, I do not see "data" as "too generic", since it's "the orientation that the image data is shipped with" - vs. "the orientation setting specified on top of that".

So, let's do `skip_reorientation` here? If the function could be named JxlDecoderSetSkipReorientation(), that would (in my view) be less ambiguous than the current name, but I see that that might not be an option here.

Reply:
Yes, `skip_reorientation` sounds good.